### DISCUSSION TITLE
Rename operation does not work for pickup options

### DISCUSSION THREAD

**Discussion by Sam115200** [Shopify/function-examples#408] [Feedback: Delivery Customization API]
Hide and Move operations work as expected for the Pickup options at the checkout. But Rename operation does not do anything, which is weird. Is this not implemented at all or is this a bug?

**Comment by javieralarcn10** [comment]
any update?

**Comment by bithaolee** [comment]
any update?

**Comment by blanklob** [comment]
@Sam115200 which hide an remove operations? Pickup options dont have move operations? can you expand more.

**Reply by Sam115200** [reply]
If you try to perform a RenameOperation on the pickup options, like change `My Standard Location` to another name, then it does not do anything.
<img width="623" alt="Screenshot 2024-09-23 at 5 54 45 PM" src="https://github.com/user-attachments/assets/31e04e8a-0c09-4cc2-b09e-c42d2742c304">

**Reply by blanklob** [reply]
Ah ok! i see
